libs/server/Resp/Objects/ListCommands.cs

```diff
@@ -380,6 +380,27 @@ private unsafe bool ListBlockingMove(RespCommand command)
             return true;
         }
 
+        /// <summary>
+        /// BRPOPLPUSH
+        /// </summary>
+        /// <returns></returns>
+        private bool ListBlockingPopPush()
+        {
+            if (parseState.Count != 3)
+            {
+                return AbortWithWrongNumberOfArguments(nameof(RespCommand.BRPOPLPUSH));
+            }
+
+            var srcKey = parseState.GetArgSliceByRef(0);
+            var dstKey = parseState.GetArgSliceByRef(1);
+            var rightOption = ArgSlice.FromPinnedSpan(CmdStrings.RIGHT);
+            var leftOption = ArgSlice.FromPinnedSpan(CmdStrings.LEFT);
+            var timeout = parseState.GetArgSliceByRef(2);
+            parseState.InitializeWithArguments(srcKey, dstKey, rightOption, leftOption, timeout);
```

Comment:
Ok I will do it, I did it in this way because there are deprecated commands and the only purpose of adding these is additioning compatibility still there commands are removed as discussed in this GitHub command https://github.com/microsoft/garnet/issues/684#issuecomment-2383617742

Reply:
I realize these are deprecated commands. Still, it's better to have good practices throughout so that other contributors can use them as reference :)

Reply:
Done